demos/python_demos/multi_camera_multi_person_tracking/multi_camera_multi_person_tracking.py

```diff
@@ -81,24 +96,32 @@ def run(params, capture, detector, reid):
         if frames is None:
             continue
 
-        all_detections = detector.get_detections(frames)
+        frame_number += 1
+        all_detections = detector.wait_and_grab()
+        detector.run_asynch(frames, frame_number)
+
         all_masks = [[] for _ in range(len(all_detections))]
         for i, detections in enumerate(all_detections):
             all_detections[i] = [det[0] for det in detections]
             all_masks[i] = [det[2] for det in detections if len(det) == 3]
 
-        tracker.process(frames, all_detections, all_masks)
+        tracker.process(prev_frames, all_detections, all_masks)
         tracked_objects = tracker.get_tracked_objects()
 
-        fps = round(1 / (time.time() - start), 1)
-        vis = visualize_multicam_detections(frames, tracked_objects, fps)
-        if not params.no_show:
```

Comment:
I think you've accidentally made it so `--no_show` has no effect. Please restore it.

Reply:
Yes, thank you. Restored.

Reply:
`waitKey` works only in a window.

Reply:
That's exactly why you should only call it if there is a window. :-)

Reply:
OK, done.